zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -637,7 +637,47 @@ paths:
             application/problem+json:
               schema:
                 $ref: "#/components/schemas/ProblemDetail"
-
+  /process-definitions/search:
+    post:
+      tags:
+        - Process definition
+      summary: process definitions (alpha)
```

```suggestion
      summary: Search process definitions (alpha)
```

:x: Summary should be `Title case`
🔧 This feels like a more complete summary of the endpoint.